nav2_behavior_tree/nav2_tree_nodes.xml

```diff
@@ -11,35 +11,35 @@
       <input_port name="backup_dist">Distance to backup</input_port>
       <input_port name="backup_speed">Speed at which to backup</input_port>
       <input_port name="time_allowance">Allowed time for reversing</input_port>
-      <input_port name="server_name">Service name</input_port>
+      <input_port name="server_name">Server name</input_port>
       <input_port name="server_timeout">Server timeout</input_port>
     </Action>
 
     <Action ID="DriveOnHeading">
       <input_port name="dist_to_travel">Distance to travel</input_port>
       <input_port name="speed">Speed at which to travel</input_port>
       <input_port name="time_allowance">Allowed time for reversing</input_port>
-      <input_port name="server_name">Service name</input_port>
+      <input_port name="server_name">Server name</input_port>
       <input_port name="server_timeout">Server timeout</input_port>
     </Action>
 
     <Action ID="CancelControl">
-      <input_port name="service_name">Service name to cancel the controller server</input_port>
+      <input_port name="server_name">Server name to cancel the controller server</input_port>
```

Comment:
This file does not change the name in the actual BT nodes, just in the representation in Groot. Are you only saying that the names are actually `server_name` in the BT nodes themselves and only the visual representation in Groot is wrong?

Reply:
I'm a little unclear from your main PR comment

Reply:
Yes, this is right. Only the representation in Groot was wrong. But the impact is that the behavior trees generated with Groot used the wrong input port names, leading to runtime errors.